ios/browser/api/ads/BUILD.gn

```diff
@@ -11,9 +11,14 @@ config("external_config") {
   include_dirs = [ "$target_gen_dir" ]
 }
 
+config("allow_shadow_variables") {
+  cflags = [ "-Wno-shadow" ]
+}
```

Comment:
Was this a warning turned error in the iOS wrapper specifically? We should probably open a separate issue for after this merges to just fix the warning properly and remove shadowing

Reply:
Fixed in 08a03ba673c7169836c91314e3d28087764661eb.